### DISCUSSION TITLE
screws_tilt_adjust for everyone

### DISCUSSION THREAD

**Discussion by mikedantelly** [Guilouz/Creality-Helper-Script-Wiki#675] [General]
Hi. First of all, I want to express my appreciation for this script and the features it provides. I realize that my situation doesn't match the list of printers the script is designed for, but maybe someone can help? 
 I am using a rooted Nebula Pad with Ender 3 v2 Neo with Sprite Extruder Pro. This printer model uses screws to adjust the table as well as BLtouch. When I add the [screws_tilt_adjust] section to printer.cfg, klipper drops the error "section 'screws_tilt_adjust' is not a valid config section". How can I fix this problem? I saw latest changelog that this option was added for KE-series printers. Does this mean that it is disabled by default for all other devices? Sorry if I'm misunderstanding something, today is my third day of dealing with Klipper and trying to set it up. I've already got my printer printing, found the right settings for printer.cfg, but I still can't get the table screw calibration to work. Is there anything I can do to fix this locally? Or does my problem require more comprehensive work on Guilouz's part that I can't even ask for.

**Comment by f1f2fedorov** [comment]
I also use a rooted Nebula Pad and an Ender 3 v2 Neo with a Sprite Extruder Pro.
I have the same problem!
Please, Guilouz, can you help us?

**Comment by LifeDay** [comment]
I think you might need to add a [bed_screws] section if it doesn't already exist.
The config here might be helpful if you didn't pull from it already:
https://github.com/Klipper3d/klipper/blob/master/config/printer-creality-ender3-v2-neo-2022.cfg

**Comment by f1f2fedorov** [comment]
Thank @LifeDay!
Ooops! 😂
I found the answer to my own question!
Thanks to @pellcorp for his tip in [discord chat ](https://discord.com/channels/1154500511777693819/1186559669855072296/1342652197107666984)
I checked and I am very pleased with the result!

**Comment by LifeDay** [comment]
@f1f2fedorov  What was the actual solution?

**Comment by f1f2fedorov** [comment]
My case is exactly the same as described by @mikedantelly above.
The [bed_screws] section was present, but it did not help.
Following @pellcorp's hint, I copied the file from https://github.com/Guilouz/Creality-Helper-Script/blob/main/files/screws-tilt-adjust/screws_tilt_adjust.py
to /usr/share/klipper/klippy/extras in the rooted Nebula Pad.
After restarting Klipper, SCREWS_TILT_CALCULATE appeared in the tools menu.
Many thanks to @Guilouz for his efforts!

**Comment by mikedantelly** [comment]
@f1f2fedorov
Awesome! Just got 1.0.0.27 pre-rooted from https://github.com/koen01/nebula_firmware and start to reconfigure my printer! Tnx!

**Comment by f1f2fedorov** [comment]
@mikedantelly 
Great!
I have been using this firmware for over a month now and I am very satisfied!
I especially like using my printer with the updated Nebula Pad - Guppy Screen shell (https://github.com/ballaswag/guppyscreen). 
You can change the root access password as described here - https://github.com/srvrguy/nebula-pad-firmware-customizer
I'm not an expert at all, but I checked and it works!
Right now, my printer is completely different from how I bought it.
